workloads/prometheus-sizing/common.sh

```diff
@@ -5,7 +5,7 @@ openshift_login
 
 export UUID=${UUID:-$(uuidgen)}
 export PROM_URL=https://$(oc get route -n openshift-monitoring prometheus-k8s -o jsonpath="{.spec.host}")
-export PROM_TOKEN=$(oc -n openshift-monitoring sa get-token prometheus-k8s)
+export PROM_TOKEN=$(oc whoami -t)
```

Comment:
`oc whoami -t` this assumes the user has an oauth token, but it's not quite equivalent if you are using a cert kubeconfig that doesn't have a token.

The replacement for `sa get-token` is `create token`. See: https://github.com/cloud-bulldozer/e2e-benchmarking/pull/454/files